compose/cli/main.py

```diff
@@ -170,6 +170,11 @@ def perform_command(self, options, handler, command_options):
         with friendly_error_message():
             handler(project, command_options)
 
+    def exit_if(self, condition, message, exit_code):
+        if condition:
+            log.error(message)
+            sys.exit(exit_code)
```

Comment:
Since this method doesn't use `self`, can we make it a module level function? There are a bunch of these at the bottom of the file.